src/markeritem.cpp

```diff
@@ -0,0 +1,52 @@
+#include "markeritem.h"
+#include <QContextMenuEvent>
```

Comment:
Clean up your includes in both this and gripitem.cpp, remove anything you're not using. (It speeds up compile times.)

Reply:
And I guess in the .h files too.